codegen/test_data/endpoints/bar_bar_method_argwithqueryparams.gogen

```diff
@@ -72,6 +73,17 @@ func (h *BarArgWithQueryParamsHandler) HandleRequest(
 	res *zanzibar.ServerHTTPResponse,
 ) {
 	var requestBody endpointsBarBar.Bar_ArgWithQueryParams_Args
+	var traceID string
+	traceID, ok := req.Header.Get("traceID")
+	if ok {
+		// only log when traceID exists
+		req.Logger.Info("Endpoint request to client",
```

Comment:
could we make this logging generated from a function, like `type converter`? So that you have more control there in the function

for zanzibar changes we should make it general, for example formatting and log content can change